test/e2e/gardener/shoot/create_authenticate_delete.go

```diff
@@ -208,7 +208,7 @@ var _ = Describe("Shoot Tests", Label("Shoot", "default"), func() {
 				g.Expect(err).ToNot(HaveOccurred())
 			}).Should(Succeed())
 
-			Expect(selfSubjectReview.Status.UserInfo.Username).To(HaveSuffix(":kubernetes-admin"))
+			Expect(selfSubjectReview.Status.UserInfo.Username).To(MatchRegexp("^.*:(kubernetes-admin|admin-user)$"))
```

Comment:
@ScheererJ Could you add a `TODO(rfranzke)` to remove this again once the non-operator setup has been removed?